lib/Conversion/TorchToMhlo/BasicOp.cpp

```diff
@@ -192,10 +189,8 @@ class ConvertAtenAddSubOp : public OpConversionPattern<AtenOpT> {
     return success();
   }
 };
-} // namespace
 
 // Binary op legalizations for Mul variants.
-namespace {
```

Comment:
The redundant anonymous namespace declaration is actually requested by LLVM coding style if I am right: https://llvm.org/docs/CodingStandards.html#anonymous-namespaces

Reply:
I got it, thanks!